sdk/communication/Azure.Communication.Common/src/EntraCommunicationTokenCredentialOptions.cs

```diff
@@ -25,23 +24,25 @@ public class EntraCommunicationTokenCredentialOptions
         /// <summary>
         /// The scopes required for the Entra user token. These scopes determine the permissions granted to the token. For example, ["https://communication.azure.com/clients/VoIP"].
         /// </summary>
-        public string[] Scopes { get; set; }
```

Comment:
I was hoping we could still keep the default and Scopes optional?

Reply:
As the identifierUris are different for both 1p apps, I think it's better to make it mandatory param, so user should specify the scopes. If you still think it makes sense to fall back to use durable identity flow by default I can update it

Reply:
And optional param for OPS might have a headache to OPS developers, as the param is optional but their flow is not working as expected

Reply:
I would prefer to have the default and optimize for the majority case